lib/ssl/src/ssl_pkix_db.erl

```diff
@@ -352,7 +352,7 @@ new_trusted_cert_entry(File, [CertsDb, RefsDb, _ | _]) ->
 add_crls([_,_,_, {_, Mapping} | _], ?NO_DIST_POINT, CRLs) ->
     [add_crls(CRL, Mapping) || CRL <- CRLs];
 add_crls([_,_,_, {Cache, Mapping} | _], Path, CRLs) ->
-    insert(Path, CRLs, Cache), 
+    [insert(Path, CRL, Cache) || CRL <- CRLs],
```

Comment:
the last element in the `CRLs` list wins?
or is it a bag?

Reply:
Good point. The table is actually a set, so this seems like the last CRL will win.  :see_no_evil: 

So the fix is not quite right then... The problem is that the point where the values in cache are used expect a single CRL, but if we insert into the cache using a key (such as the URL), it blows up because it receives a list of CRLs in that case. :thinking:

Reply:
I've changed the "fix" to flatten the CRL list just before it's iterated over [here](https://github.com/erlang/otp/blob/8c182956e1bde761ea1541fd1d061b208356cb48/lib/ssl/src/ssl_handshake.erl#L2244).

https://github.com/emqx/otp/pull/26/commits/40b6105c244992bcdbbc1bab47384e90c966b487